src/main/kotlin/app/batch/HBaseWriter.kt

```diff
@@ -142,12 +145,12 @@ class HBaseWriter : ItemWriter<DecompressedStream> {
                                     if (runMode != RUN_MODE_MANIFEST) {
                                         if (batchSizeBytes + message.length >= maxBatchVolume && batch.size > 0) {
                                             try {
-                                                logger.info("Attempting to write batch of ${batch.size} records, size $batchSizeBytes bytes to hbase with topic 'db.$database.$collection' from '$fileName'.")
+                                                logInfo(logger, "Attempting to write batch of ${batch.size} records, size $batchSizeBytes bytes to hbase with topic 'db.$database.$collection' from '$fileName'.")
                                                 putBatch(batch)
-                                                logger.info("Written batch of ${batch.size} records, size $batchSizeBytes bytes to hbase with topic 'db.$database.$collection' from '$fileName'.")
+                                                logInfo(logger, "Written batch of ${batch.size} records, size $batchSizeBytes bytes to hbase with topic 'db.$database.$collection' from '$fileName'.")
```

Comment:
I wonder if having both the pre and post logging lines is a little noisy? I can see the use in summarizing that batches are being / have been written but can't see the benefit of having both over just one?

Reply:
Also have just noticed that below we only use a post-write line (line 202) - we should bring these in line